objects/b765cf/script.lua

```diff
@@ -144,13 +144,25 @@ function PostSetup(params)
 end
 
 function englandSnap(aidBoard)
-  local snapPoints = aidBoard.getSnapPoints()
-  local newSnapPoints = {}
-  for i,v in ipairs(snapPoints) do
-    if v.rotation.y <= 269 or v.rotation.y >= 271 then
-      table.insert(newSnapPoints,v)
+    do -- Remove aidBoard snap point
```

Comment:
what is the goal of this code?

Reply:
I switched from using the rotation of the snap point to using the tag to determine which snap point is the invader discard, since that seems less happenstance.

The other part of the code is creating a snap point for the england discard. The `do ... end` blocks are so I don't need to come up with two different variable names.